.github/workflows/ci.yml

```diff
@@ -96,19 +96,28 @@ jobs:
       CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
 
   publish_pure:
+      # Build when pushing to any branch except main
+      # publish.yml will only publish if tagged ^v.*
       if: |
-        github.event_name != 'pull_request' && (
-          github.ref_name != 'main' ||
-          github.event_name == 'workflow_dispatch' )
+        (
+          github.event_name != 'pull_request' && (
+            github.ref_name != 'main' ||
```

Comment:
Why do we not run this (here and in core) when pushing to main? This whole if statement is weird.

Reply:
Moved this to the template.